assets/schema/ensemble_schema.json

```diff
@@ -1686,7 +1686,7 @@
             "uploadFiles": {
               "type": "object",
               "description": "Allow users to upload files to an API from the camera or file system",
-              "required": ["uploadApi"],
+              "required": ["uploadApi", "files"],
```

Comment:
@imjaadFRT please note the schema changes

Reply:
Do we have previous changes as well? because I am seeing bits of other changes too. 

@snehmehta please give me a gist of what's changed and where its changed. Also better if we post these changes to **schema_changes** discord channel as well.